e2e/tests/api_tests/defaultJudgments/api_dj_1v1_test.js

```diff
@@ -20,6 +20,8 @@ Scenario('Default Judgment claim', async ({I, api}) => {
   caseId = await api.getCaseId();
   await api.amendRespondent1ResponseDeadline(config.systemupdate);
   await api.defaultJudgment(config.applicantSolicitorUser);
+  await api.createFinalOrder(config.judgeUserWithRegionId1, 'FREE_FORM_ORDER');
```

Comment:
Will this break the scenarios coming after it?
The scenarios after it are dependent on the defaultJudgement command on line 22.